pkgs/fc/agent/src/fc/manage/manage.py

```diff
@@ -126,7 +126,7 @@ def check(log, enc, config: ConfigParser) -> CheckResult:
     else:
         errors.append("`nixos` channel not set.")
 
-    nixos_warnings_file = Path("/etc/fcio_nixos_warnings")
+    nixos_warnings_file = Path("/etc/fcio_nix_eval_warnings")
```

Comment:
If this file is now generated by the agent and not by the fc-nixos config, is there any reason to have this in `/etc` and not somewhere like `/var/lib` instead?

Reply:
There is no good reason, I just wanted to keep the change minimal. If you have a strong opinion on this, I can of course move the file to /var/lib/fc-agent (that directoy does not exist yet though -> yet another tmpfiles rule)